From: Scott Kitterman <scott@kitterman.com>
Date: Fri, 10 Apr 2020 23:48:07 -0400
Subject: Support RECORD files with sha256sum and file size

---
 dirtbike/strategy.py | 7 ++++++-
 1 file changed, 6 insertions(+), 1 deletion(-)

diff --git a/dirtbike/strategy.py b/dirtbike/strategy.py
index 75ef89a..16f61e8 100644
--- a/dirtbike/strategy.py
+++ b/dirtbike/strategy.py
@@ -101,7 +101,12 @@ class WheelStrategy(Strategy):
         try:
             # If we're lucky, the information for what files are installed on
             # the system are available in RECORD, aka wheel metadata.
-            files = self._metadata.get_metadata('RECORD').splitlines()
+            # RECORD may also have sha256sum and file size, we only want the
+            # name.
+            files = []
+            for line in self._metadata.get_metadata('RECORD').splitlines():
+                files.append(line.split(',')[0])
+
         # Python 3 - use FileNotFoundError
         except IOError as error:
             self._files = None
